frontend/src/pages/profile.tsx

```
@@ -32,33 +36,44 @@ export default function Profile() {
+                    height={90}
+                    width={90}
+                    alt="profile icon"
+                />
                 <div className={styles.nameText}>
                     <h3>{userData?.username}</h3>
                     <p>learner since {userData?.created_date}</p>
                 </div>
-                <button onClick={handleLogout}>
-                    Logout
-                </button>
+                <button onClick={handleLogout}>Logout</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)